scipt/create_vehicle.sh

```
@@ -0,0 +1,117 @@
+
+# Stop script on any command error
+# set -e
+
+# --- Configuration ---
+# !!! REPLACE WITH YOUR ACTUAL API BASE URL !!!
+API_BASE_URL="http://localhost:8090/api" # Example: might be http://localhost:8080
+
+# --- Existing User Credentials and UUID ---
+# !!! REPLACE WITH THE DETAILS OF AN EXISTING USER !!!
+EXISTING_USER_EMAIL="bash.user@example.com"
+EXISTING_USER_PASSWORD='Pa$$w0rd'
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)